### PR TITLE
Add a page to Export Artists

### PR DESCRIPTION
**Pull request by humor4fun**

### CODE REVIEW

File: Deemixrr/Views/Artist/Export.cshtml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+			<div class="card">
+				<div class="card-body">
+					<h5 class="card-title">CSV export - Deezer ID</h5>
+					<h6 class="card-subtitle mb-2 text-muted">Copy all of your artists' Deezer IDs into a csv file</h6>
+					<hr />
+					<div class="row">
+						<div class="col-md-12">
+							<div class="form-group">
+								<textarea class="form-control" rows="3" asp-for="ArtistDeezerIds">
+									 @foreach (var artist in Model.Artists)
```
Comment (TheUltimateC0der) [inline Deemixrr/Views/Artist/Export.cshtml]: It would be a lot easier to just use string.join()

File: Deemixrr/Views/Artist/Export.cshtml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+			<div class="card">
+				<div class="card-body">
+					<h5 class="card-title">CSV export - Artist Name</h5>
+					<h6 class="card-subtitle mb-2 text-muted">Copy all of your artists into a csv file</h6>
+					<hr />
+					<div class="row">
+						<div class="col-md-12">
+							<div class="form-group">
+								<textarea class="form-control" rows="3" asp-for="ArtistDeezerIds">
+									 @foreach (var artist in Model.Artists)
```
Comment (TheUltimateC0der) [inline Deemixrr/Views/Artist/Export.cshtml]: It would be a lot easier to just use string.join()

File: Deemixrr/Views/Artist/Export.cshtml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+			<div class="card">
+				<div class="card-body">
+					<h5 class="card-title">List export - Artist Name</h5>
+					<h6 class="card-subtitle mb-2 text-muted">Copy all of your artists into a list</h6>
+					<hr />
+					<div class="row">
+						<div class="col-md-12">
+							<div class="form-group">
+								<textarea class="form-control" rows="3" asp-for="ArtistDeezerIds">
+									 @foreach (var artist in Model.Artists)
```
Comment (TheUltimateC0der) [inline Deemixrr/Views/Artist/Export.cshtml]: It would be a lot easier to just use string.join()

**Review by TheUltimateC0der — CHANGES_REQUESTED** [review]
You also forgot to add the corresponding action to the corresponding controller

**Review by humor4fun — COMMENTED** [review]
changed the viewmodel

**Comment by humor4fun** [comment]
Missing a few components of the required changes. I'll send a new PR for you that includes those.

**Pull request closed without merging**